api/GPUDevice.json

```diff
@@ -1007,6 +1049,48 @@
             }
           }
         },
+        "optional_depthcompare_depthwriteenabled": {
+          "__compat": {
+            "description": "`depthCompare` and `depthWriteEnabled` properties are optional when not needed (for example, formats without depth).",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "120"
```

Comment:
Ditto

Reply:
Ooop, I've only got one comment from you, which says "Ditto". I'm assuming there's one missing.

Reply:
@beaufortfrancois ^

Reply:
Yes. I was wondering why `                "notes": "Currently supported on ChromeOS, macOS, and Windows only."` was not present for both.